tvm/src/template/vivado/run.tcl

```diff
@@ -10,7 +10,7 @@ set hls_prj out.prj
 open_project ${hls_prj} -reset
 
 # Top function of the design is "top"
-set_top top
```

Comment:
As you said, this is a very basic TCL script template. What I am planning to do is to add a new string stream in the code generator to provide better customization. Example: https://github.com/cornell-zhang/heterocl/blob/master/tvm/src/codegen/codegen_c.h#L230